src/core_plugins/metrics/public/components/aggs/field_select.js

```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes } from 'react';
+import _ from 'lodash';
+import Select from 'react-select';
+import AggLookup from '../lib/agg_lookup';
+import generateByTypeFilter from '../lib/generate_by_type_filter';
+
+function FieldSelect(props) {
+  const { type, fields, indexPattern } = props;
+  if (type === 'count') {
+    return null;
```

Comment:
I've seen this before in React-code, returning null in the constructor. Is there a way to avoid this?